client/src/LoginPage/LoginPage.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import '../index.css';
+import { authenticationRepository, accountRepository } from '@/_services';
+
+class LoginPage extends React.Component {
```

Comment:
Missing propTypes which is useful because you can see what props this component expects (like `history`)